packages/gatsby-theme-newrelic/README.md

```diff
@@ -2463,6 +2542,52 @@ const SearchInput = () => {
 };
 ```
 
+### `useTessen`
+
+A hook that gets allows you to instrument actions with Tessen. This hook
+requires that [`tessen`](#tessen) is configured. If `tessen` is not configured,
+calls to each action will result in a no-op.
+
+**NOTE**: Calls to the Tessen actions are pre-configured with the `nr_product`,
+and `nr_subproduct` as configured in the [`tessen`](#tessen) configuration.
+`location` is also pre-configured as `Public`. Setting these values as
+`properties` in the actions will do nothing.
+
+```js
+import { useTessen } from '@newrelic/gatsby-theme-newrelic';
+```
+
+**Arguments**
+
+none
+
+**Returns**
+
+`Tessen` - The Tessen object that allows you to call actions.
+
+**Types**
+
+```ts
+type Tessen = {
+  page: (name: string, category: string, properties?: object) => void;
+  track: (name: string, category: string, properties?: object) => void;
+};
+```
+
+**Examples**
+
+```js
+const MyComponent = () => {
+  const tessen = useUserId();
```

Comment:
```suggestion
  const tessen = useTessen();
```

Reply:
Oops good catch! Now you see my copy/paste secret 😜